src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -598,4 +598,27 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
             );
         }
     }
+
+    // -------------------------------------------------------------------------
+    // Overridden Internal Functions
+
+    /// @notice Handles issuance tokens by minting them to the receiver.
+    /// @param  _receiver The address that will receive the bought tokens.
+    /// @param  _issuanceMintAmount The amount of issuance tokens to handle.
+    function _handleIssuanceTokenAfterBuy(
+        address _receiver,
+        uint _issuanceMintAmount
+    ) internal virtual override {
+        _mint(_receiver, _issuanceMintAmount);
+    }
+
+    /// @notice Handles collateral tokens by transferring them to the receiver.
+    /// @param  _receiver The address that will receive the collateral tokens.
+    /// @param  _collateralAmount The amount of collateral tokens to handle.
+    function _handleCollateralTokensAfterSell(
+        address _receiver,
+        uint _collateralAmount
```

Comment:
Just noticed this now, but this could even just be `uint _amount`, as the function name is pretty clear in terms of what this is. Just makes it a bit simpler.

Reply:
I will just keep it. The name is descriptive and not very hard to read